src/modules/init/index.ts

```diff
@@ -88,7 +92,7 @@ const promptTemplates = async (): Promise<string> => {
 }
 
 const promptContinue = async (repoName: string) => {
-  const proceed = prop('proceed', await inquirer.prompt({
+  const proceed = prop('proceed', await enquirer.prompt({
```

Comment:
why are you using `enquirer.prompt` here instead of using `promptConfirm`?

Reply:
same applies to other similar lines

Reply:
because these are not confirmation prompts!

Reply:
Sry, this was is indeed a `confirm` prompt! Will fix it.